guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -34,16 +45,40 @@ Alternatively, to prevent downtime caused by stopping the service, you can use t
 [options="nowrap" subs="+quotes,attributes"]
 ----
 ifdef::satellite[]
```

Comment:
This ifdef should be higher. Right above "Note that this command". Also the whole block where it uses `{foreman-maintain}` is satellite-only.

Reply:
Done.